modules/potree/src/types/potree-metadata.ts

```diff
@@ -0,0 +1,70 @@
+/** Bounding box */
+export interface PotreeBoundingBox {
+  /** Min X */
+  lx: number;
+  /** Min Y */
+  ly: number;
+  /** Min Z */
+  lz: number;
+  /** Max X */
+  ux: number;
+  /** Max Y */
+  uy: number;
+  /** Max Z */
+  uz: number;
+}
+
+export type PotreeAttribute =
```

Comment:
Love the rich types. Add a TSDoc line in front of each type if possible

Reply:
Done